src/components/v2/VoteSummary/index.tsx

```diff
@@ -0,0 +1,101 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { BigNumber } from 'bignumber.js';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { generateBscScanUrl } from 'utilities';
+import { useTranslation } from 'translation';
+import { ActiveVotingProgress } from '../GovernanceProposal/ActiveVotingProgress';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { LabeledInlineContent } from '../LabeledInlineContent';
+import EllipseText from '../EllipseText';
+import { Tooltip } from '../Tooltip';
+
+import { useStyles } from './styles';
+
+type VoteFrom = {
+  address: string;
+  voteWeightWei: BigNumber;
+  comment?: string;
+};
+
+interface IVoteSummaryProps {
+  className?: string;
+  votesFrom?: VoteFrom[];
```

Comment:
I understand we are doing this because of ActiveVotingProgress but we don't need to use that component here. We can use the `LabeledProgressBar`

ActiveVotingProgress can also use the `LabledProgressBar`

Then we can simplify this to take a vote type, number of votes and total votes

Reply:
but this makes sence:

> ActiveVotingProgress can also use the LabledProgressBar

Reply:
We should cleanup this interface further. We don't need to pass all three options to every instance of this component. 
We need to pass the total for the type of vote and total for all votes and that is it.

If we do that we can swap out `ActiveVotingProgress` for `LabledProgressBar`

Reply:
I have made this update already